src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_panorama.py

```diff
@@ -560,6 +577,8 @@ def __call__(
                 A kwargs dictionary that if specified is passed along to the `AttentionProcessor` as defined under
                 `self.processor` in
                 [diffusers.cross_attention](https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/cross_attention.py).
+            circular_padding (`bool`, *optional*, defaults to `False`):
+                Whether to use circular padding for the diffusion
```

Comment:
Maybe explain a bit what circular padding is and how it helps?

Reply:
Done